examples/xoxo/xoxo.py

```diff
@@ -0,0 +1,183 @@
+from pysmt.shortcuts import FreshSymbol, Symbol, String, LE, GE, Int, Not, Or, And, Equals, Plus, Solver, StrContains
+from pysmt.typing import INT, STRING
+from enum import Enum
+import logging
+logger = logging.getLogger('xoxo')
+logger.setLevel(logging.DEBUG)
+ch = logging.StreamHandler()
+ch.setLevel(logging.INFO) # change to DEBUG for info on assertions
+logger.addHandler(ch)
+
+class Cell(Enum):
+    x = 1
+    o = 10
+    s = 0
+
+DEBUG = False
+turns = 0
+sq_size = 3
+test = 'tests/test1.txt'
+test = 'tests/blank.txt'
+
+# create symbols for the board
+board = [[FreshSymbol(INT) for _ in xrange(sq_size)]
+    for _ in xrange(sq_size)]
```

Comment:
indent this to be within the list comprehension.

Reply:
do you mean further indenting the for so it is further indented than the [[ ?

Reply:
yes, sorry that was not clear.